src/parsl-rp/parsl-rp-dag.ipynb

```diff
@@ -0,0 +1,251 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "6de66e16",
+   "metadata": {},
+   "source": [
+    "# Executing DAG Workflows with Parsl-RP\n",
+    "In this tutorial, we will execute a simple DAG (Directed Acyclic Graph) with Parsl-RP, which involves leveraging the Parsl data flow management and the Radical Pilot (RP) workload management to facilitate the concurrent execution of tasks."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "87581288-89f4-41d4-b951-b770a0956196",
+   "metadata": {},
+   "source": [
+    "<div style=\"text-align:center\"><img src=\"https://airflow.apache.org/docs/apache-airflow/stable/_images/basic-dag.png\" alt=\"my image\" width=\"30%\"></div>}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ba1877d8",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2022-11-22T12:17:48.357841Z",
+     "start_time": "2022-11-22T12:17:47.458237Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# make sure Parsl is installed\n",
+    "!pip show parsl"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "84004f91",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# make sure RCT tools are installed and up to date\n",
+    "! radical-stack"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "39d40bda",
+   "metadata": {},
+   "source": [
+    "First, let's import Parsl and RP Python modules in our application, alongside the RadicalPilotExecutor (RPEX) from Parsl"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6980a500",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import parsl\n",
+    "from parsl.config import Config\n",
+    "from parsl.app.app import python_app, bash_app\n",
+    "from parsl.executors.radical import ResourceConfig\n",
+    "from parsl.executors.radical import RadicalPilotExecutor"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "18286edc-a2b5-4f9e-974b-d0d95a627118",
+   "metadata": {},
+   "source": [
+    "Now, let's declare the `Resource Config`, which helps the RCT runtime system to determine how many resources are required for RAPTOR engine. In our case, we do not specify any and let RCT use the default (4 cores)"
```

Comment:
Review this sentence, specifically where it talks about resources (`4 cores`), so to make it clear that if user doesn't provide home much resources are needed (but in your example it is set `cores=8`), then that default value will be used.

btw, isn't default 5 cores? (`cores_per_master:1` + `worker_cores_per_node:4`)